COMETwebapp.Tests/Components/Viewer/PropertiesPanel/PropertiesComponentTestFixture.cs

```diff
@@ -98,5 +108,51 @@ public void VerifyThatComponentCanBeHidden()
             this.properties.ViewModel.IsVisible = false;
             Assert.Throws<ElementNotFoundException>(() => this.renderedComponent.Find("#properties-header"));
         }
+        
+        [Test]
+        public void VerifyElementValueChanges()
+        {
+            var compoundData = new OrderedItemList<ParameterTypeComponent>(null)
+            {
+                new ParameterTypeComponent
+                {
+                    Iid = Guid.NewGuid(),
+                    ShortName = "firstValue",
+                    Scale = new OrdinalScale
+                    {
+                        Iid = Guid.NewGuid(),
+                        ShortName = "m"
+                    },
+                    ParameterType = new SimpleQuantityKind
+                    {
+                        Iid = Guid.NewGuid(),
+                        ShortName = "m"
+                    }
+                }
+            };
+
+            var parametertype = new CompoundParameterType
+            {
+                Iid = Guid.NewGuid()
+            };
+
+            parametertype.Component.AddRange(compoundData);
+
+            var parameter = new Parameter() { Iid = Guid.NewGuid(), ParameterType = parametertype };
+
+            this.viewModel.SelectedParameter = parameter;
+
+            var compoundValues = new List<string> { "1" };
+
+            var parameterValueSet = new ParameterValueSet()
+            {
+                Iid = Guid.NewGuid(),
+                ValueSwitch = ParameterSwitchKind.MANUAL,
+                Manual = new ValueArray<string>(compoundValues),
+            };
+
+            Assert.DoesNotThrowAsync(() => this.viewModel.ParameterValueSetChanged(parameterValueSet));

```

Comment:
Please set does Assert inside an Assert.Multiple. Moreover, use the "Assert.That()"